src/olympia/addons/tasks.py

```diff
@@ -286,28 +285,23 @@ def theme_checksum(theme, **kw):
 def rereviewqueuetheme_checksum(rqt, **kw):
     """Check for possible duplicate theme images."""
     dupe_personas = Persona.objects.filter(
-        checksum=make_checksum(rqt.header_path or rqt.theme.header_path,
-                               rqt.footer_path or rqt.theme.footer_path))
+        checksum=make_checksum(rqt.header_path or rqt.theme.header_path)
+    )
     if dupe_personas.exists():
         rqt.dupe_persona = dupe_personas[0]
         rqt.save()
 
 
 @task
 @write
-def save_theme(header, footer, addon, **kw):
+def save_theme(header, addon, **kw):
     """Save theme image and calculates checksum after theme save."""
     dst_root = os.path.join(user_media_path('addons'), str(addon.id))
     header = os.path.join(settings.TMP_PATH, 'persona_header', header)
     header_dst = os.path.join(dst_root, 'header.png')
-    if footer:
-        footer = os.path.join(settings.TMP_PATH, 'persona_footer', footer)
```

Comment:
If this lands we should remove all files in that folder too (nothing you can do though)